libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/SpecTranslate/Conway.hs

```diff
@@ -798,21 +798,59 @@ instance SpecTranslate ctx (VotingProcedures era) where
               )
           <*> m
 
-instance EraPParams era => SpecTranslate ctx (GovAction era) where
+instance SpecTranslate ctx (ConwayPParams StrictMaybe era) where
+  type SpecRep (ConwayPParams StrictMaybe era) = Agda.PParamsUpdate
+
+  toSpecRep x =
+    Agda.MkPParamsUpdate
+      <$> toSpecRep (cppMinFeeA x)
+      <*> toSpecRep (cppMinFeeB x)
+      <*> pure (fmap toInteger . strictMaybeToMaybe . unTHKD $ cppMaxBBSize x)
+      <*> pure (fmap toInteger . strictMaybeToMaybe . unTHKD $ cppMaxTxSize x)
+      <*> pure (fmap toInteger . strictMaybeToMaybe . unTHKD $ cppMaxBHSize x)
+      <*> pure (fmap toInteger . strictMaybeToMaybe . unTHKD $ cppMaxValSize x)
+      <*> pure Nothing -- minUTxOValue has been deprecated and is not supported in Conway
+      <*> toSpecRep (cppPoolDeposit x)
+      <*> toSpecRep (cppKeyDeposit x)
+      <*> toSpecRep (cppEMax x)
+      <*> toSpecRep (fmap toInteger . strictMaybeToMaybe . unTHKD $ cppNOpt x)
+      <*> pure Nothing
+      <*> toSpecRep (cppPoolVotingThresholds x)
+      <*> toSpecRep (cppDRepVotingThresholds x)
+      <*> toSpecRep (cppGovActionLifetime x)
+      <*> toSpecRep (cppGovActionDeposit x)
+      <*> toSpecRep (cppDRepDeposit x)
+      <*> toSpecRep (cppDRepActivity x)
+      <*> pure (fmap toInteger . strictMaybeToMaybe . unTHKD $ cppCommitteeMinSize x)
+      <*> pure
+        (fmap (toInteger . unEpochInterval) . strictMaybeToMaybe . unTHKD $ cppCommitteeMaxTermLength x)
+      <*> toSpecRep (cppCostModels x)
+      <*> toSpecRep (cppPrices x)
+      <*> toSpecRep (cppMaxTxExUnits x)
+      <*> toSpecRep (cppMaxBlockExUnits x)
+      <*> toSpecRep (cppCoinsPerUTxOByte x)
+      <*> pure (fmap toInteger . strictMaybeToMaybe . unTHKD $ cppMaxCollateralInputs x)
```

Comment:
collateralPercentage is missing, but I can see [it's commented out in the spec](https://github.com/IntersectMBO/formal-ledger-specifications/blob/master/src/Ledger/Foreign/LedgerTypes.agda#L288C12-L288C25), so I guess we'll have to add it when it's fixed in the spec, no?

Reply:
Sounds about right. I'll resolve this to avoid this comment blocking the merge